src/aiortc/contrib/media.py

```diff
@@ -377,15 +412,19 @@ def addTrack(self, track):
             else:
                 codec_name = "aac"
             stream = self.__container.add_stream(codec_name)
-        elif track.kind == "keypoints":
-            stream = None
-        else:
+        elif (track.kind == "keypoints" and enable_prediction == True) or \
+                (track.kind == "video" and enable_prediction == False):
+            # repurpose video container stream for predicted video w/ keypoints
             if self.__container.format.name == "image2":
                 stream = self.__container.add_stream("png", rate=30)
                 stream.pix_fmt = "rgb24"
             else:
                 stream = self.__container.add_stream("libx264", rate=30)
                 stream.pix_fmt = "yuv420p"
+                stream.width = 256 #TODO add functionality ro change the width/height of the stream
```

Comment:
suggestion: can we somehow infer this from the source video and use the appropriate width/height?

Reply:
working on it, there might be a problem when the first keypoint comes before the first source video frame